specification/metrics/semantic_conventions/README.md

```diff
@@ -125,6 +126,22 @@ to the processes then to represent the count of the processes we can have a metr
 `system.processes.count`. The suffix `count` here indicates that it is the count of
 `system.processes`.
 
+This rule SHOULD only be applied to UpDownCounters, since (monotonic) Counters have
+`_total` appended to their names when they are
+[mapped to Prometheus](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/compatibility/prometheus_and_openmetrics.md#otlp-metric-points-to-prometheus),
+which would lead to `_count_total`.
+
+#### Do not use `total`
+
+Counters SHOULD NOT append `_total`  to their names. Counters have `_total` appended to their names when they are
+[mapped to Prometheus](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/compatibility/prometheus_and_openmetrics.md#otlp-metric-points-to-prometheus),
+which would lead to `_total_total`. The reason that the Prometheus mapping cannot
```

Comment:
FYI many prometheus exporters will omit _total if the metric already ends in _total, so I don't think this is a compelling reason.  IMO the better reason not to add `_total` is that it implies that the metric is cumulative, when it isn't neccessarily.